features/admin.policy-administration.v1/components/policy-administration-page-layout.scss

```
@@ -0,0 +1,27 @@
+.policy-algorithm-btn {
+    padding: 1px 16px;
+    .algorithm-txt {
+        margin-left: 4px;
+    }
+}
+
+.policy-list-card {
+    background-color: transparent;
+    padding: 0;
+}
+
+#active-policy-list-container{
```

Lets not use `ids` to identify elements.
```suggestion
#active-policy-list-container {
```

---

Addressed